### PR TITLE
0.25 docs: nested queries and custom user projections

### PR DESCRIPTION
**Pull request by mazameli**
For nested queries, I just added a blurb in the "asking questions" user guide page under the section about source data.

For remapping / custom user projections, I added a bit at the bottom of the data model section of the admin guide.

### CODE REVIEW

File: docs/users-guide/04-asking-questions.md [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+
+The first dropdown menu in the question builder is where you’ll choose the database and table you want.
+
+#### Using saved questions as source data
+
+If you've [saved some questions](06-sharing-answers.html), in the Data menu you'll see the option to use one of your saved questions as source data. What this means in practice is that you can do things like use complex SQL queries to create new tables that can be used in a question just like any other table in your database.
+
+You can use any saved question as source data, provided you have [permission](../administration-guide/05-setting-permissions.html) to view that question. You can even use questions that were saved as a chart rather than a table. The only caveat is that you can't use a saved question which itself uses a saved question as source data. (That's more inception than Metabase can handle!)
```
Comment (salsakran) [inline docs/users-guide/04-asking-questions.md]: is that true? I thought you could use any question as a starting point regardless of depth @camsaul ?

**Review by salsakran — APPROVED** [review]
Looks quite reasonable. Not sure about the maximum depth.

File: docs/users-guide/04-asking-questions.md (hunk 1 above)
Comment (salsakran) [inline docs/users-guide/04-asking-questions.md]: Asked @cam and this statement is incorrect.

Comment (camsaul) [inline docs/users-guide/04-asking-questions.md]: @sameer my GH username isn't @Cam 😂

Comment (sameer) [inline docs/users-guide/04-asking-questions.md]: you've got the wrong guy ;(

**Pull request merged**